cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -90,6 +91,66 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         }
     }
 
+    /**
+     * Translates a Python [`With`](https://docs.python.org/3/library/ast.html#ast.With) into a
+     * [Block].
+     *
+     * We return a Block to handle the with statement, following
+     * [PEP 343](https://peps.python.org/pep-0343/#specification-the-with-statement). The context
+     * managers `__enter__()` runs before the try block and `__exit__()` is always called later in
+     * the `finally` block to clean up, whether an exception occurs or not.
+     */
```

Comment:
Please add some demo example here. See https://github.com/Fraunhofer-AISEC/cpg/blob/f18321644560ea752fbbe902ed4d9b3614b9c86b/cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt#L254 for an example.